### ISSUE TITLE
Different PSNR and SSIM

### ISSUE THREAD

**Issue by samaonline** [BadourAlBahar/pose-with-style#15]
Hello, thanks for the great work! I'm trying to reproduce the PSNR and SSIM with [Global-Flow-Local-Attention](https://github.com/RenYurui/Global-Flow-Local-Attention/blob/master/PERSON_IMAGE_GENERATION.md#evaluation). With the pre-trained model, it's 17.7 and 0.70; with the images you provided [here](https://filebox.ece.vt.edu/~Badour/pose-with-style/results/posewithstyle_output.zip), it's 17.7 and 0.69. They are worse than reported in the paper.

I wonder if the difference is negligible or if the metric used is problematic? Thanks!

![metrics](https://user-images.githubusercontent.com/19239492/173973510-8b04c4c4-36fa-468a-a723-a3b1856652c7.png)

**Comment by BadourAlBahar** [comment]
I am not really sure. However, please note that we only compute the PSNR/SSIM of the foreground human. We do so using the silhouettes provided [here](https://filebox.ece.vt.edu/~Badour/pose-with-style/data/test_sil.zip).

**Comment by samaonline** [comment]
Thanks! After using the masks, the metrics roughly match.
